rio_tiler/reader.py

```diff
@@ -358,7 +359,10 @@ def point(
 
     indexes = indexes if indexes is not None else src_dst.indexes
 
-    vrt_params: Dict[str, Any] = {"add_alpha": True}
+    vrt_params: Dict[str, Any] = {
+        "add_alpha": True,
+        "resampling": Resampling[resampling_method],
```

Comment:
Note: this is not really needed because resampling should not have any effect on the `sample` command because it fetch directly the raw block. I added it so resampling_method is used somewhere in the function!